LibreTransmitter/LibreTransmitterManagerV3.swift

```diff
@@ -275,6 +276,18 @@ public final class LibreTransmitterManagerV3: CGMManager, LibreTransmitterDelega
     var lastDirectUpdate: Date?
 
     internal var countTimesWithoutData: Int = 0
+    
+    func issueTestAlert() {
```

Comment:
Errors that happen while a user is trying to use the app to do something should be presented in the current UI that is facilitating the task. System notifications should be reserved for those situations where errors are triggered from a background process, or we believe the user's attention will be elsewhere.

Reply:
issueTestAlert is not used here, I'll remove it